docs/anatomy.adoc

```diff
@@ -12,6 +12,7 @@ A substantial amount of the code in the fastai library is dedicated to reading,
 * link:../fastai/dataset.py[fastai.dataset] has a bunch of utilities for working with data arranged as files on disk. Most notably these help resize and sample data, including the splitting off of a validation set. Also noteworthy is `BaseDataset`, which extends `torch.utils.data.Dataset`, and all of its descendants. Those describe the different ways to load files arranged on disk, and a few bits of metadata like whether the output is a regression or classfication. After wrapping in a `DataLoader` the training, validation, and an optional test data set are all bundled together into a `ModelData` for use in the rest of the code.
 * link:../fastai/structured.py[fastai.structured] has methods that can massage `pd.DataFrame`s into friendlier forms for learning. This includes dealing with missing values, ensuring correct data types, normalizing numerical inputs and one-hot encoding categories, and some lightweight feature engineering on dates.
 * link:../fastai/text.py[fastai.text] has similarly intended utilities for reading and tokenizing text data.
+* link:../fastai/models[fastai.models] contains various tried and tested implementations of deep learning architectures that have been published out there. To test and attempt performance replication of a published model's weights, modules defined here can be combined with pre-trained weights by editing link:../torch_imports.py[torch_imports] to load the model and link to the published article(s) or paper(s) that describe(s) the architecture.
```

Comment:
Replication isn't really a key reason for any of this. The key reason is so that we can use transfer learning to solve problems.

Reply:
Nice to see this doc growing already :)

Should we start a new header section for modeling stuff, vs data and training? This would certainly not be the only thing there.

Reply:
That might make sense...